test/suites/ACID.m

```diff
@@ -334,6 +335,7 @@
 function [stat] = contourPenny()
   stat.description = 'Contour plot of a US\$ Penny.';
   stat.unreliable = isMATLAB('>=',[8,4]);
+  % see #604; contour() produces inconsistent output
```

Comment:
We would come back here to tackle the `unreliable` tests anyhow. But sure, I can add it. Will create a new PR, since I already merged this PR.